charts/dapr/charts/dapr_rbac/templates/ClusterRoleBinding.yaml

```diff
@@ -30,22 +30,22 @@ metadata:
   name: dapr-operator-admin
 rules:
 - apiGroups: ["*"]
-  resources: ["serviceaccounts", "deployments", "services", "configmaps", "secrets", "components", "configurations"]
+  resources: ["serviceaccounts", "deployments", "services", "configmaps", "secrets", "components", "configurations", "leases"]
```

Comment:
Why do we need "leases"?

Reply:
the kubebuilder framework used by the operator implements leader election for HA.

Reply:
In k8s > 1.18 there's an explicit need for RBAC permissions..

Reply:
can you add that as a comment in the yaml?

Reply:
I am OK with the yaml as-is.